### PR TITLE
Updated Departements.dmm and relative Icons

### PR DESCRIPTION
**Pull request by PestoVerde322**
# About The Pull Request

This PR is a QoL change for mappers, as it renew the icons for the landmarks to be more source-esque and also, changes the Departements.dmm's disposal sorting pipes names to make them more professional. Also Removed a redundant dmi file (mobs/landmarks.dmi) as all it's contents are now under effects/landmarks_statics.dmi

## Why It's Good For The Game

QoL Mapping changes good and Standardization for icons good.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

landmarkupdated

</details>

## Changelog
:cl:
fix: Adjusted Departements.dmm, now the disposal sorting pipes have more professional names and right destinations
imageadd: Renewed landmark icons for spawners
imagedel: deleted mobs/landmarks.dmi
/:cl:

### CODE REVIEW

**Comment by MarkusLarsson421** [comment]
Literally adding the Gordon Freeman spawn point placeholder. lol

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Have you manually edited the landmarks_static.dmi file to edit the job roles? That file should be entirely generated through the automatic generation tool

File: code/modules/admin/verbs/mapping.dm [hunk 1]
```diff
@@ -317,7 +317,7 @@ GLOBAL_VAR_INIT(say_disabled, FALSE)
 	//Also add the x
 	for(var/x_number in 1 to 4)
 		final.Insert(icon('icons/mob/screen_gen.dmi', "x[x_number == 1 ? "" : x_number]"), "x[x_number == 1 ? "" : x_number]")
-	fcopy(final, "icons/mob/landmarks.dmi")
+	fcopy(final, "icons/effects/landmarks_static.dmi")
```
Comment (PowerfulBacon) [inline code/modules/admin/verbs/mapping.dm:320]: The code just above this is the code that is responsible for generating the job landmark icons. They are all automatically generated and if you want to change them, you need to update this to generate the icons that you want.

Comment (PestoVerde322) [reply code/modules/admin/verbs/mapping.dm:320]: well you know something new everyday, will fix that asap

**Comment by PestoVerde322** [comment]
Closing this for the time being due to a lack of interest :/

**Pull request closed without merging**